packages/beacon-node/package.json

```diff
@@ -153,10 +157,11 @@
     "@types/jwt-simple": "0.5.33",
     "@types/leveldown": "^4.0.2",
     "@types/prometheus-gc-stats": "^0.6.1",
-    "@types/supertest": "^2.0.8",
+    "@types/supertest": "^2.0.12",
     "@types/tmp": "^0.2.0",
     "@types/varint": "^6.0.0",
     "eventsource": "^2.0.2",
+    "leveldown": "^6.1.1",
```

Comment:
oh, why is leveldown required? For libp2p datastore? @wemeetagain can you motivate upstream dep to move to level >= 8.0.0?